app/controllers/access_tokens_controller.rb

```diff
@@ -1,4 +1,5 @@
 # frozen_string_literal: true
+
```

Comment:
trailing space change?

Reply:
nah, just a newline because I added the hack here originally and then removed it :D